src/main/java/com/epam/handler/GlobalExceptionHandlerController.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.handler;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@ControllerAdvice
+@EnableWebMvc
```

Comment:
Why we need @EnbleWebMvc annotation here?

Reply:
Yes, you right, this annotation is superfluous